<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRoad</class>
 <widget class="QWidget" name="AddRoad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1120</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1120</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mini Wasalni | Add or Update Road</string>
  </property>
  <widget class="QLineEdit" name="addROAD_city1">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>230</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dosis</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
color:#BFA17E;
background: transparent;
border: 3px solid #A9724E;
border-radius: 8px; 
padding: 10px;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>Enter City-1 Name........</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addROAD_addButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>450</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Dosis</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color: #F8F8F1;
		background-color: #A9724E;
		border: none;
		border-radius: 8px;
}

QPushButton:hover {
		color: #F8F8F1;
		background-color: #BFA17E;
		border: none;
		border-radius:8px;
}

QPushButton:pressed {
		color: #F8F8F1;
		background-color: #BFA17E;
		border: none;
        border-radius: 8px;
}</string>
   </property>
   <property name="text">
    <string>ADD OR EDIT THIS ROAD</string>
   </property>
  </widget>
  <widget class="QLabel" name="addROAD_label">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>90</y>
     <width>411</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dosis</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
		color: #909178;
}</string>
   </property>
   <property name="text">
    <string>Here you can add or edit an existing road between two cities from the map. </string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="addROAD_header">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>541</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dosis</family>
     <pointsize>40</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
		color: #A9724E;
}</string>
   </property>
   <property name="text">
    <string>ADD/EDIT ROAD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Returntoupdate_addROAD">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>500</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Dosis</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color: #F8F8F1;
		background-color: #A9724E;
		border: none;
		border-radius: 8px;
}

QPushButton:hover {
		color: #F8F8F1;
		background-color: #BFA17E;
		border: none;
		border-radius:8px;
}

QPushButton:pressed {
		color: #F8F8F1;
		background-color: #BFA17E;
		border: none;
        border-radius: 8px;
}</string>
   </property>
   <property name="text">
    <string>RETURN TO UPDATE MENU</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addROAD_city2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>290</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dosis</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
color:#BFA17E;
background: transparent;
border: 3px solid #A9724E;
border-radius: 8px; 
padding: 10px;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>Enter City-2 Name........</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addROAD_distance">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>350</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dosis</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
color:#BFA17E;
background: transparent;
border: 3px solid #A9724E;
border-radius: 8px; 
padding: 10px;
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>Enter The Distance........</string>
   </property>
  </widget>
  <widget class="QLabel" name="addROAD_errorLabel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>410</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dosis</family>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
		color: #DD0505;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
